### ISSUE TITLE
Packaging on Windows fails - Keeps Downloading package '.NET Core Debugger (linux / x64)'

### ISSUE THREAD

**Issue by bigabdoul** [OmniSharp/omnisharp-vscode#1912]
## Environment data
`dotnet --info` output:
```
.NET Command Line Tools (2.0.3)

Product Information:
 Version:            2.0.3
 Commit SHA-1 hash:  12f0c7efcc

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.14393
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\2.0.3\

Microsoft .NET Core Shared Framework Host

  Version  : 2.0.3
  Build    : a9190d4a75f4a982ae4b4fa8d1a24526566c69df
VS Code version:
1.18.1
929bacba01ef658b873545e26034d1a8067445e9
```

C# Extension version: 1.13.1
## Steps to reproduce
Checkout this page: https://github.com/OmniSharp/omnisharp-vscode/wiki/Installing-the-C%23-extension-to-a-computer-without-internet-connectivity at instruction number 9:
_node node_modules/gulp/bin/gulp.js package:offline_

## Actual behavior
```
[01:48:14] Using gulpfile ~\Documents\VSCode-Projects\osvscode\omnisharp-vscode\gulpfile.js
[01:48:14] Starting 'clean'...
[01:48:14] Finished 'clean' after 11 ms
[01:48:14] Starting 'package:offline'...
[01:48:14] 'package:offline' errored after 20 ms
[01:48:14] Error: Do not build offline packages on windows. Runtime executables will not be marked executable in *nix packages.
    at doOfflinePackage (C:\Users\hp\Documents\VSCode-Projects\osvscode\omnisharp-vscode\gulpfile.js:100:15)
    at promise.then (C:\Users\hp\Documents\VSCode-Projects\osvscode\omnisharp-vscode\gulpfile.js:146:24)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:169:7)
    at Function.Module.runMain (module.js:607:11)
    at startup (bootstrap_node.js:158:16)
    at bootstrap_node.js:575:3

C:\Users\hp\Documents\VSCode-Projects\osvscode\omnisharp-vscode>node node_modules/gulp/bin/gulp.js package:offline
[01:49:03] Using gulpfile ~\Documents\VSCode-Projects\osvscode\omnisharp-vscode\gulpfile.js
[01:49:03] Starting 'clean'...
[01:49:03] Finished 'clean' after 7.27 ms
[01:49:03] Starting 'package:offline'...
Downloading package 'OmniSharp for Windows (.NET 4.6 / x64)' (20728 KB) .................... Done!
Downloading package '.NET Core Debugger (Windows / x64)' (39233 KB) .................... Done!
Installing package 'OmniSharp for Windows (.NET 4.6 / x64)'
Installing package '.NET Core Debugger (Windows / x64)'
Downloading package 'OmniSharp for OSX' (23645 KB) .................... Done!
Downloading package '.NET Core Debugger (macOS / x64)' (43968 KB) .................... Done!
Installing package 'OmniSharp for OSX'
Installing package '.NET Core Debugger (macOS / x64)'
Downloading package 'OmniSharp for Linux (x64)' (23528 KB) .................... Done!
Downloading package '.NET Core Debugger (linux / x64)' (54878 KB) .................... Done!
Installing package 'OmniSharp for Linux (x64)'
Installing package '.NET Core Debugger (linux / x64)'
Skipping package 'OmniSharp for Linux (x64)' (already downloaded).
Downloading package '.NET Core Debugger (linux / x64)' (54878 KB) .................... Done!
Installing package 'OmniSharp for Linux (x64)'
Installing package '.NET Core Debugger (linux / x64)'
Skipping package 'OmniSharp for Linux (x64)' (already downloaded).
Downloading package '.NET Core Debugger (linux / x64)' (54878 KB) .................... Done!
Installing package 'OmniSharp for Linux (x64)'
Installing package '.NET Core Debugger (linux / x64)'
Skipping package 'OmniSharp for Linux (x64)' (already downloaded).
Downloading package '.NET Core Debugger (linux / x64)' (54878 KB) .................... Done!
Installing package 'OmniSharp for Linux (x64)'
Installing package '.NET Core Debugger (linux / x64)'
Skipping package 'OmniSharp for Linux (x64)' (already downloaded).
Downloading package '.NET Core Debugger (linux / x64)' (54878 KB) .................... Done!
Installing package 'OmniSharp for Linux (x64)'
Installing package '.NET Core Debugger (linux / x64)'
Skipping package 'OmniSharp for Linux (x64)' (already downloaded).
Downloading package '.NET Core Debugger (linux / x64)' (54878 KB) .................... Done!
Installing package 'OmniSharp for Linux (x64)'
Installing package '.NET Core Debugger (linux / x64)'
Skipping package 'OmniSharp for Linux (x64)' (already downloaded).
Downloading package '.NET Core Debugger (linux / x64)' (54878 KB) ..^C
```

**Comment by rchande** [comment]
@bigabdoul You're right, this is downloading the same package in a loop. We'll investigate.

**Comment by rchande** [comment]
@akshita31 You can assign this to yourself if you want to investigate.

**Comment by akshita31** [comment]
On my system, the packaging finishes after 7 iterations for the Linux (x64) ,ie, after the line 
`Downloading package '.NET Core Debugger (linux / x64)' (54878 KB) .................... Done!`
is printed 7 times on the Console.
@bigabdoul In your output, I see you terminated it  at the 7th iteration. Do you mind checking whether it finishes after 7 iterations or continues after that as well ?

**Comment by akshita31** [comment]
My guess here is that we are performing packaging for 7 LinuxDistributions (as listed in the gulpfile.js) and so it is downloading the appropriate packages for each of them which is fine.
To avoid confusion, better print statements are required I think. Something like 
`Downloading package '.NET Core Debugger (linux / x64) Centos 7 '

Thoughts @rchande ?

**Comment by rchande** [comment]
@akshita31 Good idea. Let's do that.

**Comment by rchande** [comment]
@akshita31 Yes, I also see 7 downloads and then the script finishes.

**Comment by akshita31** [comment]
This issue has been fixed in the 1.14 milestone. We just released a beta of 1.14 - https://github.com/OmniSharp/omnisharp-vscode/releases/tag/v1.14.0-beta4. Follow [these instructions](https://github.com/OmniSharp/omnisharp-vscode/releases/tag/v1.14.0-beta4) to install a beta release of C# for VS Code.
